osu.Game/Screens/Play/Player.cs

```diff
@@ -345,9 +366,42 @@ private bool onFail()
             if (PauseOverlay.State == Visibility.Visible)
                 PauseOverlay.Hide();
 
+            // Disable input to avoid hitting objects while falling
+            GameplayClockContainer.BlockGameplayInput = true;
+
+            return true;
+        }
+
+        private void updateFail()
+        {
+            audio.Track.Frequency.Value -= Game.IsActive.Value ? 0.0015 : 0.00075;
```

Comment:
what is this `Game.IsActive` nonsense? you should be using `Clock.Elapsed` or something similar.

Reply:
Doesn't the stable version in a fail scene, when the game is not focused, it slows the failing process by half?

Reply:
How'd you figure that out?